src/main/java/com/github/wovnio/wovnjava/Settings.java

```diff
@@ -37,6 +37,11 @@
 
         String p;
 
+        p = config.getInitParameter("userToken");
+        if (p != null && p.length() > 0) {
+            this.projectToken = p;
```

Comment:
I see, thanks for the details.
Do you have a test case when userToken is set and projectToken is set with a different token? Please add one if not.